scripts/rewrite/README.md

```diff
@@ -88,18 +90,20 @@ Other configuration elements are inherited from the default `recorder` config.
 
 ### Running
 
+__Note: You should execute commands from this directory to ensure config is properly loaded.__
+
 Run every command by setting `NODE_ENV` to `rewrite-versions`.
 
 Run the script by running:
 
 ```sh
-NODE_ENV=rewrite-versions node scripts/rewrite/rewrite-versions.js
+NODE_ENV=rewrite-versions node rewrite-versions.js
```

Comment:
I would add

`cd scripts/rewrite`

to make it obvious you need to be in a different folder.
I often find this clearer than the leading Note